dds_web/scheduled_tasks.py

```diff
@@ -190,11 +190,19 @@ def delete_invite():
         try:
             invites: list = db.session.query(models.Invite).all()
             for invite in invites:
-                if (invite.created_at + timedelta(weeks=1)) < expiration:
+                invalid_invite = False
+                if invite.created_at == "0000-00-00 00:00:00":
+                    invalid_invite = True
```

Comment:
This could also be replaced with simply
```suggestion
                invalid_invite = invite.created_at == "0000-00-00 00:00:00"
```

Reply:
so this sets invalid_invite to True if equality check is true, and to False otherwise, nice 👍